spec/ioki/operator_api_spec.rb

```diff
@@ -260,15 +260,16 @@
     end
   end
 
-  describe '#drivers_set_password(product_id, driver_id, password)' do
+  describe '#update_set_password(product_id, driver_id, ...)' do
```

Comment:
```suggestion
  describe '#update_driver_password(product_id, driver_id, ...)' do
```

Should this have the same name as the method?

Reply:
yes!